packages/amplication-server/src/core/prismaSchemaUtils/schema-utils.ts

```diff
@@ -206,3 +209,35 @@ export function jsonField(field: Field) {
     return EnumDataType.Json;
   }
 }
+
+export function findFkFieldNameOnAnnotatedField(field: Field): string {
+  const relationAttribute = field.attributes?.find(
+    (attr) => attr.name === "relation"
+  );
+
+  if (!relationAttribute) {
+    throw new Error(`Missing relation attribute on field ${field.name}`);
```

Comment:
either pass the actionLog[] object to the function and add a message or catch the exception on the parent function and add a message to the log

Reply:
But at this point, you don't have access to the action log from the preparation step. We are in the convert function